nginx-autoinstall.sh

```diff
@@ -212,8 +212,9 @@ case $OPTION in
 		cd /usr/local/src/nginx/modules || exit 1
 		git clone https://github.com/google/ngx_brotli
 		cd ngx_brotli || exit 1
-		git checkout v1.0.0rc
-		git submodule update --init
+		# Use the master branch for now
+		#git checkout v1.0.0rc
+		#git submodule update --init
```

Comment:
Hi, this line need to be uncommented or:
![image](https://user-images.githubusercontent.com/14856114/140684589-3f724a31-5c11-417a-b390-f89f19f9b927.png)

Reply:
I thought I have tested it. I thought this command is only required if a checkout comment is present.